stan/math/fwd/scal/fun/log_inv_logit_diff.hpp

```diff
@@ -0,0 +1,37 @@
+#ifndef STAN_MATH_FWD_SCAL_FUN_LOG_INV_LOGIT_DIFF_HPP
+#define STAN_MATH_FWD_SCAL_FUN_LOG_INV_LOGIT_DIFF_HPP
+
+#include <stan/math/fwd/core.hpp>
+#include <stan/math/prim/scal/fun/log_inv_logit_diff.hpp>
+#include <stan/math/prim/scal/fun/inv_logit.hpp>
+#include <stan/math/prim/scal/fun/inv.hpp>
+#include <stan/math/prim/scal/fun/expm1.hpp>
+
+namespace stan {
+namespace math {
+
+using std::exp;
```

Comment:
No using statements at the top-level of namespaces.  

Could you please check that others haven't been added elsewhere?  I've seen a few of these in recent PRs.

Reply:
Ah sorry about that, I've definitely had them in some tests that I've submitted before. I'll do some searching and submit a pr to remove them.